pegasus/sites.v3/hourofcode.com/views/announcements.haml

```diff
@@ -5,10 +5,20 @@
   providers << {id: "kano", url: "https://kano.me/"}
   providers << {id: "thimble", url: "https://www.thimble.io/"}
   providers << {id: "dexter", url: "https://www.gigglebot.io/"}
+  latam_language_codes = ["la", "es", "pt", "po"]
+  show_latam_videos = DCDO.get("show_latam_videos", nil)
```

Comment:
I was thinking about removing this but I think it's still up in the air if we'll want to show the English creativity video to all audiences during MC and Dance Party launches. We can remove after HOC 2018 if we've decided it's permanent.